ts/indicators/chaikin.src.ts

```diff
@@ -0,0 +1,221 @@
+ */
+H.seriesType<Highcharts.ChaikinIndicator>(
+    'chaikin',
+    'ema',
+    /**
+     * Chaikin Oscillator. This series requires the `linkedTo` option to
+     * be set and should be loaded after the `stock/indicators/indicators.js`
+     * and `stock/indicators/ema.js`.
+     *
+     * @sample {highstock} stock/indicators/chaikin
+     *         Chaikin Oscillator
+     *
+     * @extends      plotOptions.ema
+     * @since        7.0.0
+     * @product      highstock
+     * @excluding    allAreas, colorAxis, joinBy, keys, navigatorOptions,
+     *               pointInterval, pointIntervalUnit, pointPlacement,
+     *               pointRange, pointStart, showInNavigator, stacking
+     * @optionparent plotOptions.chaikin
+     */
+    {
+        /**
+         * Paramters used in calculation of Chaikin Oscillator
+         * series points.
+         *
+         * @excluding index, period
+         */
+        params: {
+            /**
+             * The id of volume series which is mandatory.
+             * For example using OHLC data, volumeSeriesID='volume' means
+             * the indicator will be calculated using OHLC and volume values.
+             */
+            volumeSeriesID: 'volume',
+            /**
+             * Periods for Chaikin Oscillator calculations.
+             *
+             * @type    {Array<number>}
+             * @default [3, 10]
+             */
+            periods: [3, 10]
+        }
+    },
+    /**
+     * @lends Highcharts.Series#
+     */
+    {
+        nameBase: 'Chaikin Osc',
+        nameComponents: ['periods'],
+        init: function (this: Highcharts.ChaikinIndicator): void {
+            var args = arguments,
+                ctx = this;
+
+            requiredIndicator.isParentLoaded(
+                (EMA as any),
+                'ema',
+                ctx.type,
+                function (indicator: Highcharts.Indicator): undefined {
+                    indicator.prototype.init.apply(ctx, args);
+                    return;
+                }
+            );
+        },
+        getValues: function (
+            series: Highcharts.Series,
+            params: Highcharts.ChaikinIndicatorParamsOptions
+        ): (boolean|Highcharts.IndicatorValuesObject) {
+            var periods: Array<number> = (params.periods as any),
+                period: number = (params.period as any),
+                // Accumulation Distribution Line data
+                ADL: (boolean|Highcharts.IndicatorValuesObject),
+                // 0- date, 1- Chaikin Oscillator
+                CHA: Array<[number, number]> = [],
+                xData: Array<number> = [],
+                yData: Array<number> = [],
+                periodsOffset: number,
+                // Shorter Period EMA
+                SPE: (boolean|Highcharts.IndicatorValuesObject),
+                // Longer Period EMA
+                LPE: (boolean|Highcharts.IndicatorValuesObject),
+                oscillator,
+                i;
```

Comment:
`number` types are missing.

Reply:
Ouch, missed those :( Thanks for noticing. Fixed now.